R/AgeS_Computation.R

```diff
@@ -13,7 +13,7 @@
 #' \code{DATA} contains informations for more than one sample.
 #' If there is stratigraphic relations between samples, informations in DATA must be ordered by order of increasing ages.
 #' See the details section to for more informations.
-#' (2): An object of class "runjags" which is provided by the output of [AgeS_Computation]. When input of class "runjags" is identified, no new JAGS model is created. Instead, the JAGS model specified by the "runjags" object is extended. Useful for when convergence was not originally achieved and a complete restart is not desirable.
+#' (2): An object of class "BayLum.list" which is provided by the output of [AgeS_Computation]. When input of class "BayLum.list" is identified, no new JAGS model is created. Instead, the JAGS model specified by the AgeS_Computation output is extended. Useful for when convergence was not originally achieved and a complete restart is not desirable.
```

Comment:
Please use markdown syntax:
* "BayLum.list" -> \`BayLum.list\` 
* AgeS_Computation -> \[AgeS_Computation\]

The first renders the term correctly in the manual, and the second links to the function.

Reply:
Done.